stylesheets/commons/TournamentTags.less

```diff
@@ -134,6 +134,10 @@ Author(s): Elysienna
 		&.chip--tier5 {
 			background-color: @tier5-chip-background-color;
 		}
+
+		&.chip--misc {
+			background-color: @misc-chip-background-color;
```

Comment:
The var behind this, `--tournament-tag-misc-chip-background-color` is currently not defined, so it will always be the fallback. Should we add the var, or resort to `--tournament-tag-misc-badge-background-color` instead?

Reply:
@liquidely up to you

Reply:
I will add the missing color variable to the skin

Reply:
MR has been created https://gitlab.com/teamliquid-dev/liquipedia/web/skins/lakesideview/-/merge_requests/139